images/windows/hub/Dockerfile

```diff
@@ -1,7 +1,6 @@
 ARG baseImage="unityci/base:windows-latest"
 
 FROM $baseImage
-ARG hubVersion="3.0.0"
 
 # Install unity hub
-RUN choco install unity-hub --version=%hubVersion --no-progress -y
```

Comment:
Removing the version lock makes the build non-deterministic. This can be problematic, because any possible errors from version bumps should be done in development time, not during publishing time.

Reply:
Oh I moved that to my other PR #198 since its a bit more in line with the PR title. Should I move it back to this PR?

Reply:
My suggestion is to always lock the version, and keep it as a hardcoded value somewhere.

Reply:
I'll move it back then